docs/book/src/quick-start.md

```diff
@@ -14,6 +14,15 @@ This Quick Start guide will cover:
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) version v1.11.3+.
 - Access to a Kubernetes v1.11.3+ cluster.
 
+<aside class="note">
+<h1>Alternative: Use DevPod to work within a development container</h1>
+
+[![Open in DevPod!](https://devpod.sh/assets/open-in-devpod.svg)](https://devpod.sh/open#https://github.com/loft-sh/devpod-kubebuilder-template)
+
+You can use [DevPod](https://devpod.sh) and the [Kubebuilder template](https://github.com/loft-sh/devpod-kubebuilder-template) to automate the installation process by starting a development container with the needed tooling preinstalled.
```

Comment:
I think is important to mention three more things: 
- that the devpod template creates a Kubernetes cluster using [kind](https://sigs.k8s.io/kind)
- kubebuilder is already installed
- edit the `.devcontainer/Dockerfile` to update the versions of the prerequisite tools (ie golang, kubectl, kind, kubebuilder)

Reply:
Hi @csantanapr and @FabianKramm,

How to use devContainers within should not IHMO be in the quick start at all.
It is an optional/helper information. So, I think it would fit better in the FAQ section. See: https://book.kubebuilder.io/faq

 It can be How to use deploy and test your solution using GitHub workspaces? 
 
Also, we need to add a small low verbose info here to describe when/who and why do that? (i.e If you are looking for .... then you might want to ....) 

Also, do we need to use this third party? Can we not describe how to do it without this DevPod?
Can we not provide an example of the .devcontainer config file as described in: https://github.com/camilamacedo86/kubebuilder/blob/workspace/.devcontainer/devcontainer.json

Reply:
Note that, It is nice to know that this third-party can help out. However, IMO here the goal would to be to share a config without the usage of ANY third-part at all. See that we just need a config with KIND to achieve the goal or minikube and load the image locally: https://minikube.sigs.k8s.io/docs/handbook/pushing/#7-loading-directly-to-in-cluster-container-runtime

**We avoid as much as possible to relay in ANY third-party solution to provide guidance for our users.**

Reply:
@camilamacedo86 thanks for the feedback! Wouldn't the use case for this more development than testing and deployment? Sure I can move it to the FAQ, but I guess it would be more buried there.

DevPod is just an alternative to Github Codespaces, not an additional tool you need, so the config I created at ttps://github.com/loft-sh/devpod-kubebuilder-template/blob/main/.devcontainer/devcontainer.json also perfectly works with regular Github Codespaces (also solves the problem you mentioned with kind). DevPod just allows you to run this everywhere you want (including Kubernetes) and not just Github hosted servers. Its also officially mentioned on the devcontainer side at https://containers.dev/supporting#devpod